pages/index.js

```diff
@@ -1,7 +1,18 @@
-export default function HomePage() {
+import React from "react";
+import styled from "styled-components";
+import SpotCard from "../components/SpotCard.js";
+
+export default function App() {
   return (
     <div>
-      <h1>Hello from Next.js</h1>
+      <h1>TheCaliSpots</h1>
+      <SpotCard name={"Alex"} adress={"asdfsgsd"} />
     </div>
   );
 }
+
+const headline = styled.h1`
+  margin: 10px;
+  padding: 20px;
+  left: 30px;
+`;
```

Comment:
This is not used here, right?

Reply:
Yes, I tried something and honestly didn't stick to the US guidelines. I will change that in the upcoming PR. The same goes for the third point. I will make sure to pay attention to it.

Reply:
Yes, I tried something and honestly didn't stick to the US guidelines. I will change that in the upcoming PR. The same goes for the third point. I will make sure to pay attention to it.